core/memory_tracker/cc/memory_tracker.h

```diff
@@ -38,6 +40,83 @@
 namespace gapii {
 namespace TrackMemory {
 
+enum class page_protections {
+  none = 0,
+  read = 1,
+  write = 2,
+  read_write = 1 | 2
```

Comment:
I think it is still better to have them 0x0, 0x1, 0x2, 0x3?
And do we impose Google C++ style here?